tests/explicit-nulls/pos-separate/notnull/J_1.java

```diff
@@ -0,0 +1,21 @@
+package mytests.annot;
+
+public class J_1 {
+
+    @TestNotNull
+    public static final String s = "s";
```

Comment:
We also need to test an instance (non-static) field that's annotated as non-null.

Reply:
Tests added